### PR TITLE
Add LoRaCloud GLS console support

### PR DESCRIPTION
**Pull request by adriansmares**
#### Summary

References https://github.com/TheThingsNetwork/lorawan-stack/issues/2630
References https://github.com/TheThingsNetwork/lorawan-stack/pull/3932

#### Changes

- Add the LoRaCloud GLS form (contains only the token)

#### Testing

Manually setting and deleting the token for both DAS and GLS.

#### Notes for Reviewers

Screen Shot 2021-03-17 at 13 41 12
Screen Shot 2021-03-17 at 13 41 15
Screen Shot 2021-03-17 at 13 41 19

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/views/application-integrations-lora-cloud/index.js [hunk 1]
```diff
@@ -80,20 +79,26 @@ const LoRaCloud = () => {
                   path="/integrations/application-packages/lora-cloud-device-and-application-services/"
                   secondary
                 >
-                  LoRa Cloud Device & Application Services
+                  Device & Application Services
```
Comment (bafonins) [inline pkg/webui/console/views/application-integrations-lora-cloud/index.js:82]: Why dont we translate this?

File: pkg/webui/console/views/application-integrations-lora-cloud/index.js [hunk 2]
```diff
@@ -80,20 +79,26 @@ const LoRaCloud = () => {
...
-                >
+                <Link.Anchor href="https://www.loracloud.com" external secondary>
                   <Message content={m.officialLoRaCloudDocumentation} />
                 </Link.Anchor>
               </div>
               <hr className={style.hRule} />
-              <Message component="h3" content={m.setToken} />
-              <LoRaCloudDASForm />
+              <Collapse title="Geolocation" description={m.glsDescription}>
+                <>
```
Comment (bafonins) [inline pkg/webui/console/views/application-integrations-lora-cloud/index.js]: Why do you need `<Fragment />` here?

File: pkg/webui/console/views/application-integrations-lora-cloud/index.js [hunk 3]
```diff
@@ -80,20 +79,26 @@ const LoRaCloud = () => {
...
+                  <Message component="h3" content={m.setToken} />
+                  <LoRaCloudGLSForm />
+                </>
+              </Collapse>
+              <Collapse title="Device & Application Services" description={m.dasDescription}>
+                <>
```
Comment (bafonins) [inline pkg/webui/console/views/application-integrations-lora-cloud/index.js]: Why do you need `<Fragment />` here?

File: pkg/webui/console/containers/lora-cloud-gls-form/index.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    }
+  }, [appId, dispatch])
+
+  if (packageError && !isNotFoundError(packageError)) {
+    throw error
+  }
+
+  return (
+    <RequireRequest
+      requestAction={getAppPkgDefaultAssoc(appId, LORA_CLOUD_GLS.DEFAULT_PORT, selector)}
```
Comment (bafonins) [inline pkg/webui/console/containers/lora-cloud-gls-form/index.js:129]: Mb for later PRs, but I would really prefer having this fetched once the user lands on `/console/applications/{app_id}/integrations/lora-cloud`. Currently, once you expand gls/das form there is a slight delay before the form is rendered before fetching. What is more, you get this delay every time you collapse and expand the section.

**Review by bafonins — APPROVED** [review]
Looks good to me. Only small remarks.

File: pkg/webui/console/views/application-integrations-lora-cloud/index.js (hunk 1 above)
Comment (kschiffer) [reply pkg/webui/console/views/application-integrations-lora-cloud/index.js:82]: It's (sort of) a brand name of an external brand, so this should not be translated or transliterated.

File: pkg/webui/console/containers/lora-cloud-gls-form/index.js (hunk 4 above)
Comment (kschiffer) [reply pkg/webui/console/containers/lora-cloud-gls-form/index.js:129]: I agree this should be fetched on the view level, but this is not for this PR. Please open an issue.

**Pull request merged**